unuran/src/distributions/c_meixner.c

```diff
@@ -0,0 +1,252 @@
+/*****************************************************************************
+ *                                                                           *
+ *          UNURAN -- Universal Non-Uniform Random number generator          *
+ *                                                                           *
+ *****************************************************************************
+ *                                                                           *
+ *   FILE:      c_meixner.c                                                  *
+ *                                                                           *
+ *   REFERENCES:                                                             *
+ *                                                                           *
+ *****************************************************************************
+ *                                                                           *
+ *  distr: Meixner distribution                                              *
+ *                                                                           *
+ *  pdf:   f(x) = exp(beta*(x-mu)/alpha) * |Gamma(delta+ i*(x-mu)/alpha)|^2  *
+ *                                                                           *
+ *  domain:   infinity < x < infinity                                        *
+ *                                                                           *
+ *  constant: (2*cos(beta/2))^(2*delta) / (2*alpha*pi*Gamma(2*delta))        *
+ *                                                                           *
+ *  parameters: 4                                                            *
+ *     0 : alpha > 0 ... scale                                               *
+ *     1 : beta      ... shape (asymmetry) in [-pi,pi]                       *
+ *     2 : delta >0  ... shape                                               *
+ *     3 : mu        ... location                                            *
+ *                                                                           *
+ *****************************************************************************
+ *                                                                           *
+ *   Copyright (c) 2011-2012 Wolfgang Hoermann and Josef Leydold             *
+ *   Institute for Statistics and Mathematics, WU Wien, Austria              *
+ *                                                                           *
+ *   This program is free software; you can redistribute it and/or modify    *
```

Comment:
We don't use UNU.RAN distributions anywhere and also don't plan to use anytime soon. I think we can remove the distributions subdirectory.

Reply:
yes, removing the directory is an option. I think there could be useful features that one could use in SciPy, e.g., the code to sample from the generalized inverse Gaussian to speed up `geninvgauss` but there is no concrete plan to do this in the near future. I also don;t see a problem to remove the GPL header since the copyright is from Joseph and Wolfgang who asked us to use version 1.9.0 and they are aware that SciPy falls under a BSD license

Reply:
I noticed a lot of files with references to GPL (in the header comment). We can remove all of these in a follow-up PR. Since it doesn't touch the code, it shouldn't break anything on SciPy.